test/tests/ver-1ka/tests

```diff
@@ -0,0 +1,18 @@
+[Tests]
+  design = 'ODETimeDerivative.md ParsedODEKernel.md'
+  issues = '#12'
+  verification = 'ver-1ka.md'
+  [ver-1ka_csv]
+    type = CSVDiff
+    input = ver-1ka.i
+    csvdiff = ver-1ka_out.csv
+    requirement = 'The system shall be able to model a tritium volumetric source in one enclosure'
+  []
+  [ver-1ka_comparison]
+    type = RunCommand
+    command = 'python3 comparison_ver-1ka.py'
+    requirement = 'The system shall be able to generate comparison plots between the analytical solution and simulated solution of verification case 1ka, modeling a tritium volumetric source in one enclosure.'
+    prereq = ver-1ka_csv
```

Comment:
```suggestion
```
That is not needed anymore since we use the gold files to create this plot. The test above is therefore not a prerequisite for this one.